index.html

```diff
@@ -50,6 +54,41 @@
             dur: 1000; 
             loop: true">
       </a-entity>
+      <a-entity 
+        gltf-model="#red-flying-saucer" 
+        position="-1000 250 -610"
+        animation="
+          property: rotation; 
+          to: 0 360 0; 
+          loop: true; 
+          dur: 6000"
+        animation__2="
+          property: position;
+          to: 1000 250 -610;
+          loop: true;
+          dir:alternate;
+          dur: 10000;
+        ">
+      </a-entity>
+      <a-curve id="track1">
+        <a-curve-point position="1000 0 500"></a-curve-point>
+        <a-curve-point position="1000 0 -500"></a-curve-point>
+        <a-curve-point position="0 0 -1000"></a-curve-point>
+        <a-curve-point position="-1000 0 -500"></a-curve-point>
+        <a-curve-point position="-1000 0 500"></a-curve-point>
+        <a-curve-point position="-0 0 1000"></a-curve-point>
```

Comment:
There seems to be a bit of a jump when the flying saucer is behind the user, needs investigating